app/src/main/java/com/pureqml/android/ExecutionEnvironment.java

```diff
@@ -226,6 +227,8 @@ public Object invoke(V8Object v8Object, V8Array v8Array) {
                 info.add("modelName", Build.MODEL);
                 info.add("firmware", Build.VERSION.RELEASE);
                 info.add("runtime", "native");
+
+                PackageManager pm = getContext().getPackageManager();
```

Comment:
please move `PackageManager pm = getContext().getPackageManager();` in default: { } block